srcpkgs/linux5.9/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'linux5.9'
 pkgname=linux5.9
 version=5.9.13
-revision=1
+revision=2
```

Comment:
I don't think it needs a rebuild, does it? The files shouldn't change much

Reply:
I guess in any case it'll get rebuilt soon enough anyway